tf_keras/callbacks.py

```diff
@@ -3164,6 +3164,7 @@ class CSVLogger(Callback):
     Args:
         filename: Filename of the CSV file, e.g. `'run/log.csv'`.
         separator: String used to separate elements in the CSV file.
+                   Separator string ("delimiter") must be a 1-character string.
```

Comment:
Please fix indent.

Reply:
@fchollet, Fixed indent. Please check and confirm. Thank you.